### ISSUE TITLE
compilation error: 

### ISSUE THREAD

**Issue by ouboub** [gunther-bachmann/emacs-dired-annotator#2]
I just noted when byte compiling I obtain 
```
dired-annotator.el:473:22: Warning: the function ‘dired-subtree--get-ov’ is
    not known to be defined.
dired-annotator.el:306:15: Warning: the function ‘loop-until’ is not known to
    be defined.
dired-annotator.el:303:12: Warning: the function ‘measure-time’ is not known
    to be defined.
```
what are these functions? where to be found

**Comment by gunther-bachmann** [comment]
I will look into this! 

- dired-subtree is a package I use, and the code is used only if the dired-subtree package is installed. I'll have to find a way to make sure that this is compiled only if dired-subtree is installed.
- loop-until was removed in one of the later commits
- measure-time is a macro I added in a late commit, too

Thanks for the hints. I will address them (hopefully this week).

**Comment by ouboub** [comment]
>>> "GB" == Gunther Bachmann ***@***.***> writes:

> I will look into this! 
> - dired-subtree is a package I use. But this dependency should not be
>   there
> - loop-until and measure-time are (probably) some util functions I have in my emacs
>   config, again these dependencies should not be there.

> Thanks for the hints. I will address them (hopefully this week).

I downloaded loops, but measure-time, I have no idea where to find. 
However I just went over my history.

I checked out other versions in my git (well mercurial) clone, and the measure-time problem occurred when I checked out 
commit 41959858d59b86002a793

In the HEAD it seems ok. However in the meantime, I  installed so many packages, that I cannot be sure 😇

-- 
Warning: Content may be disturbing to some audiences
I strongly condemn Putin's war of aggression against the Ukraine.
I support to deliver weapons to Ukraine's military. 
I support the ban of Russia from SWIFT.
I support the EU membership of the Ukraine. 
https://addons.thunderbird.net/en-US/thunderbird/addon/gmail-conversation-view/

**Comment by gunther-bachmann** [comment]
The head should not have a problem with `loop-until` nor with `measure-time`.
The other warnings I cannot prevent for the time being. I will have to put the code for the integration with other packages into separate el-files. Then they can be used separately and the compilation should not complain. This however, I will do some time in the future. 
For now these warnings can be safely ignored.

**Comment by ouboub** [comment]
solved with commit **bd6662ca8fa5**
